lib/releaselib.sh

```diff
@@ -496,6 +496,13 @@ release::gcs::copy_release_artifacts() {
     common::sha $path 1 > "$path.sha1" || return 1
   done
 
+  logecho "- Checking whether $gcs_destination already exists..."
+  if $GSUTIL ls $gcs_destination >/dev/null 2>&1 ; then
+    logecho "- Destination exists. Assuming artifacts at destination are correct. If not, run:"
+    logecho -n "  gsutil -m rm -r $gcs_destination\n"
+    return 0
```

Comment:
Are you okay with `kubernetes-build` flaking a few times a day? We either need to:
* Squelch it here
* Add some "ignore present" flag
* Do what I did here